.gitignore

```diff
@@ -25,3 +25,7 @@ hs_err_pid*
 # IntelliJ idea
 .idea/
 *.iml
+
+#Build
+
+target
```

Comment:
Do U mind to include these, please:
# Operating System Files

*.DS_Store
.DS_Store
Thumbs.db


package.json
package-lock.json
webpack.config.js
webpack.generated.js
vscrudate.iml

# Eclipse Project Files #

.classpath
.project
.settings
.springBeans

# Test files

accessingdataneo4j.db
dependency-reduced-pom.xml
README.html